### PR TITLE
Delete cookies made optional

### PR DESCRIPTION
**Pull request by hhaamm**

### CODE REVIEW

File: seleniumbuilder/content/html/builder/record.js [hunk 1]
```diff
@@ -81,25 +81,25 @@ builder.record.stop = function() {
 
 builder.record.continueRecording = function(insertionIndex) {
   jQuery('#record-panel').show();
-  
+
```
Comment (hhaamm) [inline seleniumbuilder/content/html/builder/record.js]: A lot of "emtpy" lines with spaces.. I'm not sure why is that. My editor automatically removes the spaces from the empty lines. I hope is not a problem.

File: seleniumbuilder/content/html/gui.html [hunk 2]
```diff
@@ -60,19 +60,20 @@
...
 
       <p>
-        <form method="get" action="#record-sel2" id="startup-start-recording-sel2">
+          <form method="get" action="#record-sel2" id="startup-start-recording-sel2">
           <input type="submit" value="Record" class="button" id="startup-start-recording-sel2-button" />
         </form>
       </p>
 
       <p class="cookie-warning" id="cookie-warning"></p>
+      <input type="checkbox" id="delete-cookies" checked/>
```
Comment (hhaamm) [inline seleniumbuilder/content/html/gui.html:69]: This is the checkbox, by default is checked.

File: seleniumbuilder/content/html/builder/record.js [hunk 3]
```diff
@@ -152,11 +154,11 @@ builder.record.startRecording = function(urlText, seleniumVersion) {
     } else {
       jQuery('#heading-record').removeClass('is-on');
       if (isLoading) {
-        builder.record.recording = true;    
+        builder.record.recording = true;
         builder.gui.switchView(builder.views.script);
         builder.suite.addScript(new builder.Script(seleniumVersion));
         builder.getScript().saveRequired = true;
-        builder.getScript().addStep(new builder.Step(builder.selenium2.stepTypes.get, url.href()));
+        // builder.getScript().addStep(new builder.Step(builder.selenium2.stepTypes.get, url.href()));
```
Comment (Zarkonnen) [inline seleniumbuilder/content/html/builder/record.js]: I don't think this should be commented out?

File: seleniumbuilder/content/html/builder/record.js (hunk 1 above)
Comment (Zarkonnen) [inline seleniumbuilder/content/html/builder/record.js]: If you could fix it (along with the commented-out line below), I would be very grateful. It's very hard to see what the diff does like this.

File: seleniumbuilder/content/html/builder/record.js (hunk 3 above)
Comment (hhaamm) [inline seleniumbuilder/content/html/builder/record.js]: Ups, you are right. That shouldn't be commented out. I'll fix it.

File: seleniumbuilder/content/html/builder/record.js (hunk 1 above)
Comment (hhaamm) [inline seleniumbuilder/content/html/builder/record.js]: I removed most of the empty line changes.

File: seleniumbuilder/content/html/builder/record.js (hunk 3 above)
Comment (hhaamm) [inline seleniumbuilder/content/html/builder/record.js]: Uncommented

**Pull request merged**